### PR TITLE
Plugin - migrate tokens to secure storage

### PR DESCRIPTION
**Pull request by IKrukov-HORIS**
- tokens stored in secure storage 
- existing tokens removed from XML storage
- plugin GUI simplified

### CODE REVIEW

File: projector-plugin/src/main/kotlin/Utils.kt [hunk 1]
```diff
@@ -25,10 +22,150 @@ import java.awt.Toolkit
...
+        if (name.nodeValue == tokenName) {
+          val token = attrs.getNamedItem("value")
+          result = token.nodeValue
+          break
+        }
+      }
+    }
+    catch (e: Exception) {
+
+    }
```
Comment (SerVB) [inline projector-plugin/src/main/kotlin/Utils.kt:108]: Is it OK to ignore the exception?

File: projector-plugin/src/main/kotlin/Utils.kt [hunk 2]
```diff
@@ -25,10 +22,150 @@ import java.awt.Toolkit
...
+      }
+
+      toRemove.forEach { it.parentNode.removeChild(it) }
+
+      if (toRemove.isNotEmpty()) {
+        writeXml(doc, file)
+      }
+    }
+    catch (e: Exception) {
+    }
```
Comment (SerVB) [inline projector-plugin/src/main/kotlin/Utils.kt:141]: Same

File: projector-plugin/src/main/kotlin/Utils.kt [hunk 3]
```diff
@@ -25,10 +22,150 @@ import java.awt.Toolkit
...
+  try {
+    val factory = TransformerFactory.newInstance()
+    val transformer = factory.newTransformer()
+    transformer.setOutputProperty(OutputKeys.ENCODING, "UTF-8")
+    transformer.setOutputProperty(OutputKeys.INDENT, "yes")
+    val out = StreamResult(file)
+    transformer.transform(DOMSource(doc), out)
+  }
+  catch (e: Exception) {
+  }
```
Comment (SerVB) [inline projector-plugin/src/main/kotlin/Utils.kt:170]: Same

File: projector-plugin/src/main/kotlin/Utils.kt (hunk 1 above)
Comment (IKrukov-HORIS) [reply projector-plugin/src/main/kotlin/Utils.kt]: Fixed

File: projector-plugin/src/main/kotlin/Utils.kt (hunk 2 above)
Comment (IKrukov-HORIS) [reply projector-plugin/src/main/kotlin/Utils.kt]: Fixed

File: projector-plugin/src/main/kotlin/Utils.kt (hunk 3 above)
Comment (IKrukov-HORIS) [reply projector-plugin/src/main/kotlin/Utils.kt]: Fixed

**Pull request merged**